### PR TITLE
concatenate the source files

### PR DESCRIPTION
**Pull request by trungpham**

### CODE REVIEW

**Comment by trungpham** [comment]
https://github.com/ericf/grunt-css-selectors/issues/3

**Comment by trungpham** [comment]
@ericf is it okay to merge this in?

File: tasks/css_selectors.js [hunk 1]
```diff
@@ -33,10 +33,10 @@ module.exports = function (grunt) {
                         return css.use(selectors.replace(m.search, m.replace));
                     }
                 });
-
-                grunt.file.write(fileGroup.dest, css.toString(options));
-                grunt.log.writeln('File "' + fileGroup.dest + '" created.');
+                return css.toString(options); 
             });
+            grunt.file.write(fileGroup.dest, cssString.join(require('os').EOL));
```
Comment (ericf) [inline tasks/css_selectors.js]: Can you move this `require()` to the top of the file with the other ones?

**Comment by ericf** [comment]
@trungpham Yeah, thanks! Can you add a test for this too?

**Comment by trungpham** [comment]
@ericf test added.

**Pull request merged**